packages/framer-motion/src/render/dom/DOMVisualElement.ts

```diff
@@ -37,4 +38,21 @@ export abstract class DOMVisualElement<
     }
 
     KeyframeResolver = DOMKeyframesResolver
+
+    childSubscription?: VoidFunction
+    handleChildMotionValue() {
+        if (this.childSubscription) {
+            this.childSubscription()
+            delete this.childSubscription
+        }
+
+        const { children } = this.props
```

Comment:
It might still be worth exploring why a `children` prop is set on the SVG elements.

<img width="468" alt="image" src="https://github.com/user-attachments/assets/76688082-2b63-487f-a6ba-81c4ad434104">

Screenshot from example in Chrome macOS: http://localhost:9990/?example=SVG-Text-MotionValue-Child